lib/generators/blacklight/templates/catalog_controller.rb

```diff
@@ -180,6 +180,10 @@ class <%= controller_name.classify %>Controller < ApplicationController
     # If there are more than this many search results, no spelling ("did you 
     # mean") suggestion is offered.
     config.spell_max = 5
+
+    # Configuration for autocomplete suggestor
+    config.autocomplete_enabled = true
```

Comment:
Yeah I thought about this, but I left with the specified enabled, because you may have this defined but not enabled? The configuration felt different from enabling it. Not feeling too strongly about this, but seems more in your face way to disable it.